hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationScheduler.java

```diff
@@ -505,22 +508,26 @@ private void processPriorityMessages() {
             }
         }
 
-        @edu.umd.cs.findbugs.annotations.SuppressWarnings({"VO_VOLATILE_INCREMENT" })
+        @edu.umd.cs.findbugs.annotations.SuppressWarnings({"VO_VOLATILE_INCREMENT"})
         private void process(Object task) {
```

Comment:
Hmm, we wouldn't get rid of any existing try-catch-block and the whole method will be indented once more. Not sure if that will really improve the readability.